Source/AliveLib/Glukkon.cpp

```diff
@@ -252,7 +252,7 @@ void Glukkon::M_Walk_1_442D30()
         case 10:
         case 14:
             PlaySound_4447D0(0, this);
-            field_212++;
+            currentWalkPitch_212++;

```

Comment:
rename to be field_212_currentWalkPitch all of the "field_addr" will be removed by a script at the end